### ISSUE TITLE
Microprimes don't stake

### ISSUE THREAD

**Issue by techcoincommunity** [PaycoinFoundation/paycoin#471]
Microprimes stopped staking after the rate decrease...haven't had a stake payout since 1/15.

**Comment by IngCr3at1on** [comment]
Yeah... for some reason no-one notified me on that one for awhile. Should be fixed by https://github.com/PaycoinFoundation/paycoin/pull/469 but it's a forking commit so I can't just merge it at will.
